temporal-sdk/src/main/java/io/temporal/activity/LocalActivityOptions.java

```diff
@@ -120,17 +128,17 @@ public Builder mergeActivityOptions(LocalActivityOptions override) {
     }
 
     /**
-     * RetryOptions that define how activity is retried in case of failure. Default is null which is
-     * no retries.
+     * {@link RetryOptions} that define how an Activity is retried in case of failure.
+     * Default is null which is no retries.
```

Comment:
I don't think this is true any more?
@mfateev shouldn't this change to something like "Activities use a default Retry Policy if these Retry options are not provided" ?

Reply:
Ya looks like it's returning `DEFAULT_INSTANCE` with some basic initialized values. I've updated this.
https://github.com/temporalio/sdk-java/blob/master/temporal-sdk/src/main/java/io/temporal/common/RetryOptions.java#L57